compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -470,6 +477,41 @@ private void defineConstructs(BLangPackage pkgNode, SymbolEnv pkgEnv) {
         }
     }
 
+    private void populateTypeToTypeDefMap(List<BLangNode> typeDefNodes) {
+        for (BLangNode typeDef : typeDefNodes) {
+            if (typeDef.getKind() == NodeKind.CLASS_DEFN) {
+                continue;
+            } else if (typeDef.getKind() == NodeKind.TYPE_DEFINITION) {
+                BLangTypeDefinition typeDefNode = (BLangTypeDefinition) typeDef;
+                BType type = typeDefNode.typeNode.getBType();
+                typeToTypeDef.put(type, typeDefNode);
+            }
```

Comment:
Can we not simply check if `typeDef.getKind() != NodeKind.TYPE_DEFITION` then continue and do the map insertion?